src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -50,11 +58,15 @@ def __init__(self, uta_db: UtaDatabase, mane_transcript: ManeTranscript) -> None
         >>> result.genomic_data.start, result.genomic_data.end
         (156864428, 156881456)
 
+        :param: seqrepo_access: SeqRepo instance to give access to query SeqRepo database
         :param uta_db: UtaDatabase instance to give access to query UTA database
         :param mane_transcript: Instance to align to MANE or compatible representation
+        :param mane_transcript_mappings: Instance to provide access to ManeTranscriptMappings class
         """
+        self.sr = seqrepo_access
```

Comment:
Update in other places in this class

```suggestion
        self.seqrepo_access = seqrepo_access
```